pkg/lib/serverTime.js

```diff
@@ -258,42 +266,43 @@
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_os_release() {
+        if (self.os_release)
+            return Promise.resolve(self.os_release);
+        else
+            return read_os_release().then(release => {
+                self.os_release = release;
+                return release;
+            });
+    }
+
+    function get_timesync_backend() {
+        if (self.backend)
+            return Promise.resolve(self.backend);
+        else
+            return get_os_release().then(release => {
+                const config = cockpit.manifests.base1.config;
+                let backend = "none";
+                if (release &&
+                    release.ID &&
+                    config &&
+                    config["timesync-backend"])
+                    backend = config["timesync-backend"][release.ID];
+                self.backend = backend;
+                return backend;
+            });
+    }
+
+    function get_custom_ntp_timesyncd() {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
 
-    self.get_custom_ntp = function () {
-        /* We only support editing the configuration of
-         * systemd-timesyncd, by dropping a file into
-         * /etc/systemd/timesyncd.conf.d.  We assume that timesyncd is
-         * used when:
-         *
-         * - systemd-timedated is answering for
-         *   org.freedesktop.timedate1 as opposed to, say, timedatex.
-         *
-         * - systemd-timesyncd is enabled (false if chrony is being used)
-         *
-         * The better alternative would be to have an API in
-         * o.fd.timedate1 for managing the list of NTP server
-         * candidates.
-         */
         const result = {
-            supported: false,
+            supported: true,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20221213-115634-adc42487-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)